cmd/server/assets/login/resetpassword.html

```diff
@@ -0,0 +1,81 @@
+{{define "login/resetpassword"}}
+<!doctype html>
+<html lang="en">
+
+<head>
+  {{template "floatingform" .}}
+  {{template "head" .}}
+  {{template "firebase" .}}
+</head>
+
+<body class="bg-light">
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <div class="d-flex vh-100">
+      <div class="d-flex w-100 justify-content-center align-self-center">
+        <div class="col-sm-6">
+          <div class="card shadow-sm">
+            <div class="card-header">Reset password</div>
+            <div class="card-body">
+              <form id="loginForm" class="floating-form" action="/" method="POST">
+                <div class="form-label-group mb-2">
+                  <input type="email" id="email" name="email" class="form-control" placeholder="Email address" required
+                    autofocus />
+                  <label for="email">Email address</label>
+                </div>
+
+                <button type="submit" id="submit" class="btn btn-primary btn-block">Send reset email</button>
+              </form>
+            </div>
+            <div class="card-body">
+              <a class="card-link" href="/">&larr; Login</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </main>
+
+  {{template "scripts" .}}
+  {{template "loginscripts" .}}
+
+  <script type="text/javascript">
+    $(function() {
+      let $form = $('#loginForm');
+      let $submit = $('#submit');
+      let $email = $('#email');
+
+      $form.on('submit', function(event) {
+        event.preventDefault();
+
+        if (window.confirm("Are you sure you want to reset your password?") !== true) {
+          return;
+        }
+
+        let email = $email.val();
+
+        // Disable the submit button so we only attempt once.
+        $submit.prop('disabled', true);
+
+        firebase.auth().sendPasswordResetEmail(email).then(function() {
+          clearExistingFlash();
+          flash("Password reset email sent.", "success");
+          window.setTimeout(function() {
```

Comment:
Let's not redirect - we dont want them to miss the flash message, and there's a convenient back button on the page already

Reply:
I was debating this to myself.... removed